microsoft-edge/devtools-guide-chromium/whats-new/2023/08/devtools-116.md

```diff
@@ -0,0 +1,115 @@
+---
+title: What's new in DevTools (Microsoft Edge 116)
+description: "Move the location of the Activity Bar in Focus Mode. React Developer Tools extensions works in Focus Mode.  Containment Diff view in the Memory tool when multiple heap snapshots.  And more."
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 08/01/2023
+---
+# What's New in DevTools (Microsoft Edge 116)
+
+<!-- todo: ms.date above: put date of 116 Stable -->
+
+[!INCLUDE [Microsoft Edge team note for top of What's New](../../includes/edge-whats-new-note.md)]
+
+
+<!-- ====================================================================== -->
+## "Move Activity Bar" button in the Activity bar
+
+<!-- Subtitle: __ -->
+<!-- todo: need Subtitle sentence -->
+
+While in Focus Mode, you now have the ability to easily move the Activity bar's location with a click of a button.  In the upper left of DevTools, to the right of the **Inspect** button and **Device Emulation** button, there's now a **Move Activity Bar to left** or **Move Activity Bar to top** button.  The new button is always available in the Activity bar, instead of having to go into the DevTools **Settings** (...) menu.
+
+For example, by default, the Activity bar's orientation is horizontal, but you can change it to vertical by simply clicking the new **Move Activity Bar to left** button:
+
+![Moving the location of the Activity Bar in Focus Mode](./devtools-116-images/move-location-activity-bar.gif)
+
+See also:
+* [What's New entries about Focus Mode](/search/?scope=Microsoft%20Edge&terms=focus%20mode)
+<!--
+* [Dec. 2023 blog post about Focus Mode]() - todo: link -->
+
+
+<!-- ====================================================================== -->
+## React Developer Tools extension retains state of tools in Focus Mode
+
+<!-- Subtitle: The React Developer Tools extension retains the state of tools while in Focus Mode. -->
```

Comment:
add Subtitle sentence for entry: React Developer Tools extension retains state of tools in Focus Mode
@leahmsft

Reply:
won't do since we won't add to welcome tool